cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -116,12 +117,16 @@ public Conversion(CgmesModel cgmes, Conversion.Config config, List<CgmesImportPo
         this(cgmes, config, postProcessors, NetworkFactory.findDefault());
     }
 
-    public Conversion(CgmesModel cgmes, Conversion.Config config, List<CgmesImportPostProcessor> postProcessors,
-                      NetworkFactory networkFactory) {
+    public Conversion(CgmesModel cgmes, Conversion.Config config, List<CgmesImportPostProcessor> postProcessors, NetworkFactory networkFactory) {
+        this(cgmes, config, postProcessors, networkFactory, null);
```

Comment:
By default, use `Reporter.NO_OP`

Reply:
used no-operation reporter instead of null